LibTrixi.jl/Project.toml

```diff
@@ -3,5 +3,10 @@ uuid = "7e097bd5-a775-4bc1-90b6-ad92bd7220c1"
 authors = ["Michael Schlottke-Lakemper <michael@sloede.com>", "Benedict Geihe <bgeihe@uni-koeln.de>"]
 version = "0.1.0"
 
+[deps]
+OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+Trixi = "a7f1ee26-1774-49b1-8366-f1abc58fbfcb"
```

Comment:
These all require a corresponding `[compat]` entry, similar to
https://github.com/trixi-framework/Trixi.jl/blob/bea4bfed27ce896edcfa3dbe79861a4faef4dec7/Project.toml#L51-L53